quipucords/scanner/network/connect_callback.py

```diff
@@ -32,14 +33,16 @@ class ConnectResultCallback(CallbackBase):
     scan, as we scan it.
     """
 
-    def __init__(self, result_store, credential, display=None):
+    def __init__(self, result_store, credential, scan_task, display=None):
```

Comment:
Are you finding any value in capturing the connect data? I hesitate to collect this since it has no input on the actual facts.

Reply:
I captured it for completeness, but it might be interesting if we have a scan that's not connecting to a particular host to see if the connect scan status was failed or unreachable. I don't feel strongly about this if you want to get rid of it, though.

Reply:
Reviewing the log. I don't see much value in this, I'd skip it for now.

Reply:
Will do.

Reply:
Were you going to pull this out?